src/ConnectedMode/UI/ConnectionInfoComponent.xaml

```diff
@@ -21,25 +23,25 @@
             <ColumnDefinition />
         </Grid.ColumnDefinitions>
 
-        <Image Grid.Column="0" Height="30" Margin="{Binding ElementName=ConnectionInfoUc, Path= ImageMargin}" 
-               VerticalAlignment="{Binding ElementName=ConnectionInfoUc, Path= TextAndIconVerticalAlignment}" HorizontalAlignment="Center">
+        <Image Grid.Column="0" Height="25" Margin="{Binding ElementName=ConnectionInfoUc, Path= ImageMargin}"
```

Comment:
Some spacing here as well, and maybe a bit smaller icon
![image](https://github.com/user-attachments/assets/41c24e9b-380d-4b64-a2cf-a63d283075b3)